components/public-api-server/pkg/apiv1/workspace_test.go

```diff
@@ -195,7 +195,7 @@ func TestWorkspaceService_ListWorkspaces(t *testing.T) {
 
 	connPool := &FakeServerConnPool{}
 
-	route, handler := v1connect.NewWorkspacesServiceHandler(NewWorkspaceService(connPool))
+	route, handler := v1connect.NewWorkspacesServiceHandler(NewWorkspaceService(connPool), connect.WithInterceptors(auth.NewServerInterceptor()))
```

Comment:
Is it worth adding a function which will construct these handlers with the interceptor, rather than specifying it manually each time?

Reply:
Yeah, can do. I did that for the TeamsService, but the existing WorkspaceService require more refactoring to streamline them. I can follow up on this.

Reply:
Sorry, I meant I can follow-up on that in a different PR. If you do want to see it here, I can do that also.

Reply:
ah, ok. Different PR is fine 👍